tests/tests_geomstats/test_multivariate_normal.py

```diff
@@ -26,21 +68,95 @@ def test_sample(self, n, point, n_samples, expected):
         self.assertAllClose(self.Space(n).sample(point, n_samples).shape, expected)
 
     def test_point_to_pdf(self, n, point, n_samples):
-        point = gs.to_ndarray(point, to_ndim=2, axis=0)
-        Space = self.Space(n)
-        samples = Space.sample(point[0, :], n_samples)
-        samples = gs.to_ndarray(samples, to_ndim=2, axis=0)
+        space = self.Space(n)
+        samples = space.sample(space.random_point(), n_samples)
+        result = space.point_to_pdf(point)(samples)
 
+        samples = gs.to_ndarray(samples, to_ndim=2, axis=0)
+        point = gs.to_ndarray(point, to_ndim=2, axis=0)
         expected = []
         for i in range(point.shape[0]):
-            loc, cov = Space._unstack_location_diagonal(n, point[i, ...])
+            loc, cov = space._unstack_location_diagonal(n, point[i, ...])
```

Comment:
@alebrigant is the function called `reformat` the same as this `_unstack_location_diagonal` ?

Reply:
the function `reformat` is also not tested.

Reply:
> @alebrigant is the function called `reformat` the same as this `_unstack_location_diagonal` ?

No not exactly.